README.md

```diff
@@ -45,17 +45,33 @@ with more configuration options.
 From within the root of the repo run the following command to build an image of
 `renterd` tagged `renterd`.
 
-```bash
-docker build -t renterd .
+### Production
+
+```sh
+docker build --build-arg BUILD_TAGS='netgo' -t renterd:latest -f ./docker/Dockerfile .
+```
+
+### Testnet
+
+```sh
+docker build --build-arg BUILD_TAGS='netgo testnet' -t renterd-testnet:latest -f ./docker/Dockerfile .
 ```
 
 ### Run Container
 
 Run `renterd` in the background as a container named `renterd` that exposes its
 API to the host system and the gateway to the world.
 
+### Production
+
+```bash
+docker run -d --name renterd -e RENTERD_API_PASSWORD="<PASSWORD>" -e RENTERD_SEED="<SEED>" -p 127.0.0.1:9980:9980/tcp -p :9981:9981/tcp ghcr.io/siafoundation/renterd:latest
```

Comment:
I think the images need to be 
`ghcr.io/siafoundation/renterd:master`
`ghcr.io/siafoundation/renterd:master-zen`
since we updated that recently to match `hostd`.

@peterjan can you confirm that? If so we probably should get rid of the old tags to avoid confusion.

Reply:
I can confirm. Best to stay as consistent as possible.

Reply:
i updated the image names in the readme